5-WebApp-AuthZ/5-2-Groups/WebApp-OpenIDConnect-DotNet.csproj

```diff
@@ -19,11 +19,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Graph" Version="1.21.0" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="..\..\Microsoft.Identity.Web.UI\Microsoft.Identity.Web.UI.csproj" />
-    <ProjectReference Include="..\..\Microsoft.Identity.Web\Microsoft.Identity.Web.csproj" />
+    <PackageReference Include="Microsoft.Identity.Web.UI" Version="0.1.0-localbuild" />
```

Comment:
Oh, I understand Microsoft.Identityu.Web.UI also references Microsoft.Identity.Web ...

Reply:
fixing these. thanks.

---
In reply to: [406348152](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/pull/331#discussion_r406348152) [](ancestors = 406348152)